### PR TITLE
added vehicle command for position control submodes

### PR DESCRIPTION
**Pull request by tumbili**
the new command introduces submodes for manual position control mode.
examples are circle mode or zoom out.

Signed-off-by: Roman <bapstr@ethz.ch>

### CODE REVIEW

**Comment by tumbili** [comment]
@LorenzMeier Can you have a look at this?

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -1298,6 +1298,22 @@
                   <param index="1">The target VTOL state, as defined by ENUM MAV_VTOL_STATE. Only MAV_VTOL_STATE_MC and MAV_VTOL_STATE_FW can be used.</param>
               </entry>
 
+              <entry value="4000" name="MAV_CMD_SET_POSCTRL_SUBMODE_STANDARD">
```
Comment (LorenzMeier) [inline message_definitions/v1.0/common.xml]: Can you replace POSCTRL with "Guided", which is MAVLink standard terminology?

File: message_definitions/v1.0/common.xml [hunk 2]
```diff
@@ -1298,6 +1298,22 @@
...
+                  <description>This command sets the submode to standard position control when vehicle is in position control mode. The vehicle holds position and altitude and the user can input the desired velocites along all three axes.
+                  </description>
+              </entry>
+
+              <entry value="4001" name="MAV_CMD_SET_POSCTRL_SUBMODE_CIRCLE">
```
Comment (LorenzMeier) [inline message_definitions/v1.0/common.xml]: Same here.

**Comment by LorenzMeier** [comment]
Looking good.

**Pull request merged**